.github/workflows/build.yml

```diff
@@ -0,0 +1,45 @@
+name: Build and push docker image
+
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: philanthropydatacommons/service
+
+on:
+  push:
+    branches:
+      - 'main'
+
+jobs:
+  build-and-push-docker-image:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v4
+      - run: echo VERSION="$(git log -1 --date=unix --pretty=format:'%cd' | date --utc '+%Y%m%d')"-"$(git log -1 --pretty=format:'%h')" >> "$GITHUB_ENV"
+      - name: Build with node
+        uses: actions/setup-node@v4
+        with:
+          node-version: 22.13.0
```

Comment:
Not trying to belabor the patch vs major point, but this build step can be on `22` rather than specifying an older version, right? (I know it isn't older now, but it will be soon)

Reply:
Also good catch, I can make it match the other flows here, again despite my better judgment.